lib/components/user/sequential-pane-display.js

```diff
@@ -61,12 +63,12 @@ class SequentialPaneDisplay extends Component {
           <FormNavigationButtons
             backButton={prevId && {
               onClick: this._handleToPrevPane,
-              text: 'Back'
+              text: <FormattedMessage id='common.forms.back' />
             }}
             okayButton={{
               disabled: disableNext,
               onClick: this._handleToNextPane,
-              text: nextId ? 'Next' : 'Finish',
+              text: <FormattedMessage id={`common.forms.${nextId ? 'next' : 'finish'}`} />,
```

Comment:
Please do not create dynamic message IDs as it is hard to automatically keep track of which IDs are in the code. Instead, please create a hardcoded message ID string for each possible message.